README.md

```diff
@@ -57,6 +57,7 @@ There are several configuration settings supported:
 | `signWithParams`      | No       | Params to pass to signtool.  Overrides `certificateFile` and `certificatePassword`. |
 | `iconUrl`             | No       | A URL to an ICO file to use as the application icon (displayed in Control Panel > Programs and Features). Defaults to the Atom icon. |
 | `setupIcon`           | No       | The ICO file to use as the icon for the generated Setup.exe |
+| `skipUpdateIcon`      | No       | When set to true, prevents setupIcon-related error messages "This application could not be started" that may occur during installation. |
```

Comment:
This isn't actually what the `skipUpdateIcon` option does, it's the _effect_ of enabling the option but it doesn't actually do this.  A more appropriate thing to put here would be

```
Disables setting the icon of `Update.exe`, this can solve startup errors if you are building your application on macOS.
```

Reply:
I'm all for having more accurate documentation, and indeed I focused on the indirect effect rather than the direct one.

However, I never ran/built anything on macOS, so I am not sure that your description is entirely accurate either? Because for sure I needed to enable that option.